tests/client/amo/sagas/testIndex.js

```diff
@@ -0,0 +1,15 @@
+import { fork } from 'redux-saga/effects';
+
+import rootSagas from 'amo/sagas';
+import categoriesSaga from 'amo/sagas/categories';
+
+
+describe('amo rootSagas', () => {
+  it('should get Api from state then make API request to categories', () => {
+    const sagaGenerator = rootSagas();
+
+    assert.deepEqual(sagaGenerator.next().value, [
+      fork(categoriesSaga),
```

Comment:
Hmm, this test doesn't seem so useful. It looks like it will be hard to maintain, especially as sagas grow. Maybe you could just call `sagaGenerator.next()` to make sure no errors are thrown?